admin_manual/configuration_server/index.rst

```diff
@@ -1,6 +1,6 @@
-=============================
-ownCloud Server Configuration
-=============================
+====================
+Server Configuration
```

Comment:
Thanks for the feedback. But, I feel that saying it is redundant, as the reader should know that they're reading ownCloud documentation.